### ISSUE TITLE
[terra-application-layout] Missing themeable variables

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-framework#1169]
# Bug Report

The following themeable variables are missing:

**cerner-clinical-theme**

--terra-application-layout-demo-extensions-color - packages/terra-application-layout/lib/terra-dev-site/test/common/demoStyles.module.scss (line: 3, column: 4)

**orion-fusion-theme**

--terra-application-layout-demo-extensions-color - packages/terra-application-layout/lib/terra-dev-site/test/common/demoStyles.module.scss (line: 3, column: 4)
